Dockerfile.db_clean

```diff
@@ -1,4 +1,4 @@
-FROM flyway/flyway:7.8.2-alpine
+FROM flyway/flyway:9.10.1-alpine
```

Comment:
Do we want to lock this to a patch version? `flyway/flyway:9.10` will pull the latest patch version. We _could_ also do the same for minor version: `flyway/flyway:9`. It's a bit more risky but it _should_ be backwards compatible

Reply:
I could go either way. I'd probably rather do patch, since I saw in the release notes for Flyway that sometimes breaking changes are released in Minor versions 😬

Reply:
I'm good with patch

Reply:
Sweet. I'll update to [9.10-alpine](https://hub.docker.com/layers/flyway/flyway/9.10-alpine/images/sha256-07f2358b0f75c8bcdf1b07deb5853ccf21152bb012c261f0f60151128638b008?context=explore)